inc/refresh/debug.h

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright (C) 1997-2001 Id Software, Inc.
```

Comment:
Please use current year and your name in the copyright headers for new files.

Reply:
Sure.
It's how it came from upstream and I didn't pay much mind to it.
I extended the notices with my name, and to give proper credit, the others I amalgamated the changes from.